packages/teams-js/src/public/pages.ts

```diff
@@ -374,12 +377,14 @@ export namespace pages {
       registerHandler('settings.remove', handleRemove, false);
     }
 
+    const setValidityStateLogger = configLogger.extend('setValidityState');
     /**
      * Sets the validity state for the configuration.
      * The initial value is false, so the user cannot save the configuration until this is called with true.
      * @param validityState - Indicates whether the save or remove button is enabled for the user.
      */
     export function setValidityState(validityState: boolean): void {
+      setValidityStateLogger('setting validity state to %i', validityState);
       ensureInitialized(FrameContexts.settings, FrameContexts.remove);
       if (!isSupported()) {
         throw errorNotSupportedOnPlatform;
```

Comment:
>throw errorNotSupportedOnPlatform; [](http://example.com/codeflow?start=8&length=34)

Based on the logging you added, I think that we could infer what happens when looking at logs but maybe since we are interested in this very specific issue it would make sense to also add a statement here that just says something like "pages.config not supported, setValiditityState throwing errorNotSupportedOnPlatform" to make it very obvious?

Reply:
no prob